### ISSUE TITLE
Mega ticket to move taxonomy work forward

### ISSUE THREAD

**Issue by mmazanec22** [codeforamerica/brigade-project-index-statusboard#69]
(Break up this ticket-- send via email, make tickets, etc)
- Chris to move his modularization PR forward for project index
- Ray to work on library that categorizes tags into taxonomy (using gitsheets)
- Someone to use that library in the index so that indexed projects also have taxonomy tags
- Use that library in this project for tag search
- In parallel: work on taxonomy itself
- Front end for anyone to make PRs to taxonomy-- see tags, make suggestions
- Taxonomy repo needs documentation on all the ways to access the taxonomy-- backend free ways to use the data-- let front end devs use taxonomy directly

**Comment by mmazanec22** [comment]
Made this ticket during a meeting to capture all the work that needs done on taxonomy.  Conversation has been moved to an email thread.  Closing in favor of scoped tickets in appropriate repos that will come as a result of that thread.
